pom.xml

```diff
@@ -16,8 +16,21 @@
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
+    <license.licenseName>MIT</license.licenseName>
   </properties>
 
+  <developers>
+    <developer>
+      <name>Eclipse Trace Compass Community</name>
+      <url>eclipse.dev/tracecompass</url>
+    </developer>
+  </developers>
+  <licenses>
+    <license>
+       <name>MIT License</name>
+       <url>https://opensource.org/license/MIT</url>
+    </license>
+  </licenses>
```

Comment:
Can you please move the elements description, scm and issueManagement below here to have all metadata together in the pom?